laterpay/views/frontend/partials/widget/explanatory-overlay.php

```diff
@@ -36,7 +36,23 @@
                 <?php
                     // ignoring this because generated html is escaped in,
                     // views/frontend/partials/widget/purchase-button.php
-                    echo $overlay_data['action']; // phpcs:ignore
+                echo wp_kses( $overlay_data['action_html_escaped'], [
```

Comment:
Since we know the output is escaped and I changed the key name, will it work without using `wp_kses` here? @tomjn

Reply:
It should do but since the wp_kses call is already here i see no harm in keeping it anyway